examples/colab/wav2vec2_saved_model_finetuning.ipynb

```diff
@@ -0,0 +1,2441 @@
+{
```

Comment:
<p>nit: in <em>the</em> LibriSpeech directory. Let's explore these files.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>